src/components/Modals/ENSMapAddressModal/ENSMapAddressModal.tsx

```diff
@@ -0,0 +1,61 @@
+import { useCallback, useEffect, useState } from 'react'
+import { Button, Close, Field, Message } from 'decentraland-ui'
+import Modal from 'decentraland-dapps/dist/containers/Modal'
+import ethereumImg from '../../../icons/ethereum.svg'
+import { Props } from './ENSMapAddressModal.types'
+import styles from './ENSMapAddressModal.module.css'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+
+export default function EnsMapAddressModal(props: Props) {
+  const { isLoading, error, onClose, onSave } = props
+  const [address, setAddress] = useState<string>('')
+
+  const handleSave = useCallback(() => {
+    onSave(address)
+  }, [address, onSave])
+
+  const handleAddressChange = useCallback(
+    (_evt, { value }) => {
+      setAddress(value)
+    },
+    [setAddress]
+  )
+
+  useEffect(() => {
+    document.getElementById('address-input')?.focus()
+  }, [])
+
+  return (
+    <Modal closeIcon={<Close />} onClose={onClose} size="tiny">
+      <div className={styles.main}>
+        <div className={styles.info}>
+          <h1 className={styles.title}>{t('ens_map_address_modal.title')}</h1>
+          <span className={styles.description}>{t('ens_map_address_modal.description')}</span>
+        </div>
+        <div>
+          <Field
+            id="address-input"
```

Comment:
What do you think about adding the type `address` to the Field component?
```suggestion
            id="address-input"
            type="address"
```